docs/teamshares/recipes/buttons.md

```diff
@@ -0,0 +1,190 @@
+# How to convert SharedUI::Button usages to Shoelace
+
+## 1. Link button
+
+> 💡 Hint: these are buttons marked with `external: true` or marked with `render_as_link: true`
+
+### Before
+
+```
+= render SharedUI::ButtonComponent.new(\
+  text: "Schedule prep session",
+  link: "https://www.calendly.com/mickmoylan",
+  icon: "calendar_icon",
+  external: true,
+  method: "get",
+  size: "medium",
+  type: "outlined")
+```
+
+### After
+
+```
+sl-button[size="large" pill href="https://www.calendly.com/mickmoylan" target="_blank"]
```

Comment:
For the shoelace examples, I'd prefer to put the individual attributes on their own lines to encourage readability:
```
sl-button[
  size="large" 
  pill href="https://www.calendly.com/mickmoylan" 
  target="_blank"
]
```